packages/pyright-internal/src/tests/samples/reportAny.py

```diff
@@ -17,4 +17,7 @@ def baz() -> None: ...
 
 match(bar):
     case _:
-        ...
\ No newline at end of file
+        ...
+
+cast(int, bar)
+cast(int, bar.asdf)
```

Comment:
so is this just ignoring the final expression, intermediate expressions still report?

Reply:
yea, and i think that's fine (and also consistent with how basedmypy behaves)